src/components/DragAndDrop/DropZoneViewHolder.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
Wrong file naming: 

The folder should contain the component name and then create index files for platforms. Please follow the same for all of the components.

Reply:
The idea is to have drag and drop related components together. I guess i can make one parent folder and then for each component a child folder

Reply:
Yup child folder works.